app.py

```diff
@@ -68,6 +68,14 @@
     _LOGGER.info("installation=%s", RUNTIME_CONTEXT.app_installation)
 
 
+# We simply extend the GitHub Event set for our use case ;)
+@process_event("thoth_thamos_advise", action="finished")
+@process_webhook_payload
+async def on_advise_workflow_finished(*, action, analysis_id, repo_full_name, head_sha, installation, **kwargs):
```

Comment:
`repo_full_name`: Argo uses a complete URL to a repository (so that it could handle multiple git scms), so perhaps use `repo_url` instead as well?

Also, I would call it `on_thamos_workflow_finished` so that we don't confuse it with the adviser workflow, what do you think?

Reply:
so we turn it into `repo_url` (the machine-readable url, not the human-readable html url), and `fetch_ref_spec` which is more specific that argo's revision?

Reply:
How about we discard both of these and pass just `check_run_id`? (Then the workflow is passed along with repo and revision also the `check_run_id`?

https://developer.github.com/v3/checks/runs/#update-a-check-run